examples/allowed_memory.rs

```diff
@@ -0,0 +1,54 @@
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::unreadable_literal))]
+
+extern crate elf;
+use std::{iter::FromIterator, ptr::addr_of};
+
+extern crate rbpf;
+
+// The following example uses an ELF file that was compiled from the ebpf-allowed-memory.rs file
+// It is built using the [aya framework](https://aya-rs.dev/).
+// Once the aya dependencies (rust-nightly, latest llvm and latest bpf-linker) are installed, it
```

Comment:
rust-nightly will change, it might be more relevant to specify a version number instead?

Reply:
Aya itself relies on nightly: https://aya-rs.dev/book/start/development/
The example doesn't use anything fancy and is unlikely to break.
If we put a specific nightly version here, we'll have to fix all deps, including llvm (since that needs to match your nightly version), bpf-linker and aya. This has a higher likelihood of becoming stale/hard to install properly.